packages/jbrowse-web/src/configuration/index.js

```diff
@@ -43,15 +43,45 @@ function getModelConfig(tree) {
   return tree
 }
 
+/**
+ * helper method for readConfObject, reads the config from a mst model
+ *
+ * @param {object} model instance of ConfigurationSchema
+ * @param {array} slotPaths array of paths to read
+ * @param {any} args extra arguments e.g. for a feature callback, will be sent to each of the slotNames
+ */
 function getConf(model, slotName, args) {
   if (!model.configuration)
     throw new Error(`cannot getConf on this model, it has no configuration`)
   return readConfObject(model.configuration, slotName, args)
 }
-
+/**
+ * helper method, applies getConf to an array of slotNames
+ *
+ * @param {object} model instance of ConfigurationSchema
+ * @param {array} slotNames array of config paths to read
+ * @param {any} args extra arguments e.g. for a feature callback, will be sent to each of the slotNames
+ */
+function getConfs(model, slotNames, args) {
+  return slotNames.map(slotName => getConf(model, slotName, args))
+}
+/**
+ * helper method, applies readConfObject to an array of slotNames
+ *
+ * @param {object} model instance of mst model with a configuration object
+ * @param {array} slotNames array of config paths to read
+ * @param {any} args extra arguments e.g. for a feature callback, will be sent to each of the slotNames
+ */
+function readConfObjects(confObject, slotNames, args) {
```

Comment:
👍

Reply:
I wasn't ultra happy with how this looked when used but it could be useful in some places